### PR TITLE
Fix regis

### PR DESCRIPTION
**Pull request by RayWP**
fix docker, and remove stacktrace, replace with simple err.out(), later will implement proper beautiful logging and sentry

### CODE REVIEW

File: registration/Dockerfile.old [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java17-22.1.0 AS builder
```
Comment (aldy505) [inline registration/Dockerfile.old:1]: delete this file

File: registration/.gitignore [hunk 2]
```diff
@@ -1,9 +1,11 @@
 HELP.md
-target/
+target/*
+!target/*.jar
```
Comment (aldy505) [inline registration/.gitignore:3]: no. don't do this.

File: registration/Dockerfile [hunk 3]
```diff
@@ -1,32 +1,5 @@
...
-
-RUN yum update -y && yum install -y curl
-
-# Add Spring Boot Native app pesto to Container
-COPY --from=builder "/build/target/pesto" pesto
-
-# Fire up our Spring Boot Native app by default
-CMD [ "sh", "-c", "./pesto -Dserver.port=$PORT" ]
\ No newline at end of file
+ADD target/pesto-registration.jar pesto-registration.jar
```
Comment (aldy505) [inline registration/Dockerfile:3]: no. don't do this. the jar file must NOT exists on the repository. build the jar file inside the dockerfile instead.

**Review by aldy505 — CHANGES_REQUESTED** [review]

File: registration/Dockerfile (hunk 3 above)
Comment (RayWP) [reply registration/Dockerfile]: done

File: registration/.gitignore (hunk 2 above)
Comment (RayWP) [reply registration/.gitignore:3]: done

File: registration/Dockerfile.old (hunk 1 above)
Comment (RayWP) [reply registration/Dockerfile.old]: done

**Review by aldy505 — APPROVED** [review]

**Pull request merged**